### PR TITLE
LPS-156677

### PR DESCRIPTION
**Pull request by Natanaelbenvindo**
https://issues.liferay.com/browse/LPS-156677

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase [hunk 1]
```diff
@@ -823,6 +823,87 @@ definition {
 		ObjectPortlet.viewEntry(entry = "Entry B");
 	}
 
+	@description = "LPS-151676 - Verify that it's possible to edit the relationships of the Custom Object entry (One-to-Many)"
+	@priority = "4"
+	test CanEditOneToManyRelationshipOfCustomObjectEntries {
+		task ("Add a Custom Object, a field, create a 1toM relationship with the User object, and add an entry (via API)") {
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:829]: Given some Custom Object entries that contain a Custom Field and Relationship (One-to-many)

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase [hunk 2]
```diff
@@ -823,6 +823,87 @@ definition {
...
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 293",
+				objectName = "CustomObject293",
+				pluralLabelName = "Custom Objects 293");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:833]: To Custom Object, add the number ticket correct. Would be: 
`Custom Object 156677`
`CustomObject156677`
`Custom Objects 156677`

Comment (andreefarias) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:833]: After change to all Custom Objects

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase [hunk 3]
```diff
@@ -823,6 +823,87 @@ definition {
...
+
+				ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject293");
+			}
+
+			task ("Add Entrys via API") {
+				for (var fieldEntry : list "A,B,C,D") {
+					ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+						fieldName = "customObjectField",
+						objectName = "CustomObject293",
+						value = "Entry${fieldEntry}");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:869]: You can use space on value to entries.

Comment (andreefarias) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:869]: You will need replace all.

**Review by andreefarias — CHANGES_REQUESTED** [review]
Hey @Natanaelbenvindo,

Good work. I request some changes to better readability.

**Comment by andreefarias** [comment]
ci:test:sf

**Review by andreefarias — APPROVED** [review]

**Comment by andreefarias** [comment]
Thanks for the changes. Good work! LGTM! 🎉

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase [hunk 4]
```diff
@@ -823,6 +823,89 @@ definition {
...
+
+		task ("Then the Custom Object entry is updated") {
+			for (var fieldEntry : list "B,C") {
+				ObjectAdmin.goToCustomObject(objectName = "CustomObject156677");
+
+				ObjectPortlet.viewEntryDetails(entry = "Entry ${fieldEntry}");
+
+				ObjectPortlet.selectRelationshipFieldEntry(
+					entry = "Entry A",
+					newEntry = "Entry D");
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:898]: This should be the When. "When the relationship is edited"

**Review by epark18 — COMMENTED** [review]
Hey @Natanaelbenvindo great test, I have some suggestions on mainly the tasks so if you need any additional context please let me know. Thank you!

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase [hunk 5]
```diff
@@ -844,6 +844,84 @@ definition {
...
+		task ("When a Custom Object entry is edited") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject156677");
+
+			ObjectPortlet.viewEntryDetails(entry = "Entry B");
+
+			ObjectPortlet.selectRelationshipFieldEntry(
+				entry = "Entry A",
+				newEntry = "Entry C");
+
+			task ("Then the Custom Object entry is updated") {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:916]: Can you fix the spacing on this task? It should be in line with Given and When

**Comment by epark18** [comment]
@Natanaelbenvindo good job, it just needs that spacing fix and will be good to go. Thank you!

**Comment by epark18** [comment]
ci:forward

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase [hunk 6]
```diff
@@ -844,6 +844,84 @@ definition {
...
+			ObjectPortlet.viewEntryDetails(entry = "Entry B");
+
+			ObjectPortlet.selectRelationshipFieldEntry(
+				entry = "Entry A",
+				newEntry = "Entry C");
+
+		task ("Then the Custom Object entry is updated") {
+				CreateObject.assertEntrySelectedOnRelationshipField(entry = "Entry C");
+
+				PortletEntry.save();
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:919]: @Natanaelbenvindo these are spaced too far. Could you run SF please? thank you!

Comment (Natanaelbenvindo) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase:919]: ok, i'll fix it

Comment (Natanaelbenvindo) [reply modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase]: done

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**